operations/helm/tests/metamonitoring-values-generated/mimir-distributed/templates/metamonitoring/mixin-alerts.yaml

```diff
@@ -631,9 +631,9 @@ spec:
           }} has not shipped any block in the last 4 hours.
         runbook_url: https://grafana.com/docs/mimir/latest/operators-guide/mimir-runbooks/#mimiringesterhasnotshippedblocks
       expr: |
-        (min by(cluster, namespace, pod) (time() - thanos_shipper_last_successful_upload_time) > 60 * 60 * 4)
+        (min by(cluster, namespace, pod) (time() - cortex_ingester_shipper_last_successful_upload_time) > 60 * 60 * 4)
```

Comment:
Does this work? With two ingesters, one that's uploading regularly and other one that only uploaded in the past and not since, I think this will never trigger. Say first ingester reports time "now - 10min", and second reports "now - 5h". Then `time() - cortex_ingester_shipper_last_successful_upload_time` will return `10min` and `5h`, and `min` of that is `10m`.

Reply:
Oh, it's `min by (pod)`.